docker/runners/runner-monitor.sh

```
@@ -53,29 +81,62 @@ slack_post() {
+    -H "Authorization: Bearer ${RUNNER_GITHUB_TOKEN}" \
+    -H "Accept: application/vnd.github+json" \
+    "https://api.github.com/orgs/${RUNNER_ORG}/actions/runners?per_page=100" 2>/dev/null || true)
 
-# Check for missing runners (expected but not even in docker ps)
-for i in $(seq 1 $EXPECTED_RUNNERS); do
-    name="omninode-runner-${i}"
-    if [[ -z "${current_status[$name]+x}" ]]; then
+if [[ -z "${github_json}" ]]; then
+    unhealthy_list+=("GITHUB_API: failed to fetch org runner status")
+else
+    while IFS=$'\t' read -r name status busy; do
```

Shellcheck (0.11.0)

[warning] busy appears unused. Verify use (or export if used externally).

(SC2034)